pkg/gui/controllers/branches_controller.go

```diff
@@ -117,6 +117,11 @@ func (self *BranchesController) GetKeybindings(opts types.KeybindingsOpts) []*ty
 			Tooltip:           self.c.Tr.MergeBranchTooltip,
 			DisplayOnScreen:   true,
 		},
+		{
+			Key:         opts.GetKey(opts.Config.Branches.SquashBranch),
+			Handler:     opts.Guards.OutsideFilterMode(self.squash),
```

Comment:
What about using `GetDisabledReson` here? This operation does not make sense with multiple items/multiple items handling is not implemented yet.
```suggestion
			Handler:     opts.Guards.OutsideFilterMode(self.squash),
			GetDisabledReason: self.require(self.singleItemSelected()),
```